src/Aquifer.API/Endpoints/Projects/Start/Endpoint.cs

```diff
@@ -53,6 +58,8 @@ public override async Task HandleAsync(Request request, CancellationToken ct)
 
         await dbContext.SaveChangesAsync(ct);
 
+        await notificationService.SendProjectStartedNotificationAsync(project.Id, ct);
```

Comment:
I think we should just remove this for now, because the post-processing queue for the project start is going to handle this.

Reply:
Is there a strong objection?  I'd ideally like to ship all of the email notifications at the same time.  After we do the translation work then this endpoint will kick off a different queue and that queue will kick off the notification.  It's only a few lines of code moving around.